cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
+\descript{\href{https://www.ucf.edu/degree/information-technology-bs/}{B.S. in Information Technology}}
```

Comment:
(JC):
Did you do the Secure Computing and Networks minor? If you do, probably not worth
mentioning given how long ago and all your great experience since.